tests/foreman/ui/test_subscription.py

```diff
@@ -337,20 +337,32 @@ def test_select_customizable_columns_uncheck_and_checks_all_checkboxes(session):
     """
     checkbox_dict = {
         'Name': False,
-        'Type': False,
         'SKU': False,
         'Contract': False,
         'Start Date': False,
         'End Date': False,
         'Requires Virt-Who': False,
+        'Type': False,
         'Consumed': False,
         'Entitlements': False
     }
+    org = entities.Organization().create()
+    _, temporary_local_manifest_path = mkstemp(prefix='manifest-', suffix='.zip')
+    with manifests.clone() as manifest:
+        with open(temporary_local_manifest_path, 'wb') as file_handler:
```

Comment:
Can't you just write to `manifest.filename`?

Reply:
I just copied the code from here https://github.com/SatelliteQE/robottelo/pull/7188/files#diff-a500afc8cabfe510bb5e7e9f3f8faa73R87.

Reply:
I think we should use `manifest.filename`, or even extend this manifests to write file for us, but yeah, let's leave it as it is.